src/main/java/com/google/sps/data/ComputeAPIClient.java

```diff
@@ -0,0 +1,243 @@
+        }
+        else if (operation == "stopInstance") {
+          op = stopInstance(instanceName);
+        }
+        else if (operation == "restartInstance") {
+          op = restartInstance(instanceName, sessionId.get());
+        }
+        else {
+          System.out.println("Operation not found!");
+        }
+        // Call Compute Engine API operation and poll for operation completion 
+        // status
+        System.out.println("Waiting for operation completion...");
+        Operation.Error error = blockUntilComplete
+          (compute, op, OPERATION_TIMEOUT_MILLIS);
+        if (error == null) {
+          System.out.println("Success!");
+        } else {
+          System.out.println(error.toPrettyString());
+        }
+      } catch (IOException e) {
+          System.out.println("Got error!");
+          System.err.println(e.getMessage());
+      } catch (Throwable t) {
+          t.printStackTrace();
+        }
+    }
+    
+  // [START create_instances]
+  public Operation createInstance(String instanceName, String sessionId)
+    throws IOException {
+      Instance instance = new Instance();
+      instance.setName(instanceName);
+      instance.setMachineType(String.format(
+        "projects/taniecer-step-2020/zones/us-central1-f/machineTypes/n1-standard-1",
+        PROJECT_ID, ZONE_NAME));
+      NetworkInterface ifc = new NetworkInterface();
+      ifc.setNetwork(String.format(
+        "https://www.googleapis.com/compute/v1/projects/%s/global/networks/default",
+        PROJECT_ID));
+      List<AccessConfig> configs = new ArrayList<>();
+      AccessConfig config = new AccessConfig();
+      config.setType(NETWORK_INTERFACE_CONFIG);
+      config.setName(NETWORK_ACCESS_CONFIG);
+      configs.add(config);
+      ifc.setAccessConfigs(configs);
+      instance.setNetworkInterfaces(Collections.singletonList(ifc));
+      AttachedDisk disk = new AttachedDisk();
+      disk.setBoot(true);
+      disk.setAutoDelete(true);
+      disk.setType("PERSISTENT");
+      AttachedDiskInitializeParams params = new AttachedDiskInitializeParams();
+      params.setDiskName(instanceName);
+      params.setSourceImage(SOURCE_IMAGE_PREFIX + SOURCE_IMAGE_PATH);
+      params.setDiskType(String.format(
+        "projects/taniecer-step-2020/zones/us-central1-a/diskTypes/pd-standard",
+        PROJECT_ID, ZONE_NAME));
+      disk.setInitializeParams(params);
+      instance.setDisks(Collections.singletonList(disk));
+      // Adds startup script to be used by VM Instance.
+      Metadata meta = new Metadata();
+      Metadata.Items item = new Metadata.Items();
+      item.setKey("startup-script-url");
+      item.setValue(String.format("gs://%s/starter_script.sh", PROJECT_ID));
+      meta.setItems(Collections.singletonList(item));
+      Metadata.Items item2 = new Metadata.Items();
+      item2.setKey("session-id");
+      item2.setValue(sessionId);
+      meta.setItems(Collections.singletonList(item2));
```

Comment:
setItems as a collections list of both of the items, you are overriding it both times